### PR TITLE
Add setPedFightingStyle for clientside peds

### PR DESCRIPTION
**Pull request by StrixG**

### CODE REVIEW

**Comment by qaisjp** [comment]
Please revert edc9180

It should return false on failure. Only getters return nil on failure.

**Review by qaisjp — DISMISSED** [review]
Please revert edc9180

It should return false on failure. Only getters return nil on failure.

**Comment by botder** [comment]
Wait for the result of #821 before reverting, if required.

File: Client/mods/deathmatch/logic/luadefs/CLuaPedDefs.cpp [hunk 1]
```diff
@@ -1700,6 +1702,25 @@ int CLuaPedDefs::SetPedDoingGangDriveby(lua_State* luaVM)
...
+{
+    CClientEntity*   pEntity;
+    unsigned char    ucStyle;
+    CScriptArgReader argStream(luaVM);
+    argStream.ReadUserData(pEntity);
+    argStream.ReadNumber(ucStyle);
+
+    if (argStream.HasErrors())
+    {
+        m_pScriptDebugging->LogCustom(luaVM, argStream.GetFullErrorMessage());
```
Comment (patrikjuvonen) [inline Client/mods/deathmatch/logic/luadefs/CLuaPedDefs.cpp]: The contents of this if-block should be changed to:
```cpp
return luaL_error(luaVM, argStream.GetFullErrorMessage());
```

**Review by patrikjuvonen — CHANGES_REQUESTED** [review]

**Review by patrikjuvonen — APPROVED** [review]
Thank you! All ready to go once 1.5.7 is released.

**Pull request merged**